en/flight_modes_fw/land.md

```diff
@@ -14,7 +14,7 @@ Where possible, instead use [Return mode](../flight_modes/return.md) with a pred
 
 :::note
 
-- This mode requires a valid global position estimate (from GPS or inferred from a [local position](../ros/external_position_estimation.md#enabling-auto-modes-with-a-local-position)).
+- This mode requires a valid local position estimate.
```

Comment:
@sfuhrer As per https://discuss.px4.io/t/do-really-all-autonomous-modes-require-global-position/34890/20?u=hamishwillee I suspect land requires a position estimate that can be either local or global.

As it reads now, you could have a global position and as a reader I would not think it would work. My assumption here is that it requires a valid position estimate, either local or global. 

So what modes do you really require a global position for? Mission and return I presume because that specifies locations in terms of GPS. But what about orbit mode? That requires a GPS position, so how can it work with only a local position?

Lastly, consider https://docs.px4.io/main/en/config/safety.html#position-gps-loss-failsafe - I believe this to be correct - it reflects position as either local or global and that is fine. Right?

Reply:
>be either local or global.

There is no global position without local position, so the text could read "requires at least local position estimate"

[Here](https://github.com/PX4/PX4-Autopilot/blob/44d1003f8e4d1c1ed1d8c7a3d6f8f623b43a8fe4/src/modules/commander/ModeUtil/mode_requirements.cpp#L137-L142) is where the mode requirements are set in PX4.
Those modes do only require local position:
NAVIGATION_STATE_AUTO_TAKEOFF
NAVIGATION_STATE_AUTO_LAND
NAVIGATION_STATE_AUTO_FOLLOW_TARGET
NAVIGATION_STATE_AUTO_PRECLAND
NAVIGATION_STATE_ORBIT
NAVIGATION_STATE_AUTO_VTOL_TAKEOFF

And for those you need global:
NAVIGATION_STATE_AUTO_MISSION
NAVIGATION_STATE_AUTO_LOITER
NAVIGATION_STATE_AUTO_RTL

>Lastly, consider https://docs.px4.io/main/en/config/safety.html#position-gps-loss-failsafe - I believe this to be correct - it reflects position as either local or global and that is fine. Right?

Yes, that's fine, as in the failsafe case you usually lose local and global position estimate at the same time. Local vs global mode requirements only really matter for mode acceptance (which modes you are able to fly indoors for example).